### PR TITLE
Fixes Create account padding issue

### LINKED ISSUE #33978: [Safari][MacOS][iOS] "Create Account" links have uneven padding
### **Description**
On Safari (MacOS and iOS) the "Create an account" links do not have even padding on the top/bottom.  

### **User Story**
**As a** VA.gov Product Owner
**I want** the padding for the "Create Account" link to be even
**So that** user have a consistent experience across devices/OS's

### **Repro Steps**
1.) Visit va.gov using Safari on MacOS or iOS
2.) Click "Sign In"
3.) On the modal, observe the "Create an account with ID.me" text

**What happens:** Padding is not uniform vertically
**What should happen**:  Padding should be uniform

### **Acceptance Criteria**

- [ ] Padding is uniform across OS/Device/Browser

### **Notes**
Issue was reported by @codyreinold-usds .  Full text below:
_in Safari on MacOS and iOS, the "Create an account" links do not have even padding on top / bottom as they do on non-Mac / iOS devices. padding should be equal_.

### PR DESCRIPTION
**Pull request by asg5704**
## Description
This PR fixes an issue that was reported by OCTO in which they mentioned _Safari on MacOS and iOS, the "Create an account" links do not have even padding on the top / bottom as they do on non-Mac/iOS devices.  The padding should be equal throughout and on all screen sizes._

## Original issue(s)
Closes department-of-veterans-affairs/va.gov-team#33978

## Testing done
Visual

## Screenshots
Original (Safari)

Current (Safari)
Safari

Current (Chrome)
Chrome

## Acceptance criteria
- [x] Padding is equal on Safari
- [x] Padding is equal on other Browsers

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/platform/site-wide/user-nav/sass/user-nav.scss [hunk 1]
```diff
@@ -323,7 +323,9 @@ span.sidelines {
 #create-account a {
   border-top: 1px solid $color-gray-light;
   border-bottom: 1px solid $color-gray-light;
-  padding-bottom: 1.16em;
+  padding: 1.16em 0;
```
Comment (timwright12) [inline src/platform/site-wide/user-nav/sass/user-nav.scss:326]: Could this use the 32px design system utility `vads-u-padding-y--4` instead? cc @k80bowman

File: src/platform/site-wide/user-nav/sass/user-nav.scss [hunk 2]
```diff
@@ -323,7 +323,9 @@ span.sidelines {
...
+  line-height: 100%;
+  width: 100%;
```
Comment (timwright12) [inline src/platform/site-wide/user-nav/sass/user-nav.scss:328]: We have a design system class we should use here instead: `vads-u-width--full` cc @k80bowman  

https://design.va.gov/utilities/height-and-width

File: src/platform/site-wide/user-nav/sass/user-nav.scss (hunk 1 above)
Comment (AGarcia-Clarity) [reply src/platform/site-wide/user-nav/sass/user-nav.scss:326]: Changing to `vads-u-padding-y--3` 🚀

**Review by Kwoly — APPROVED** [review]

**Comment by AGarcia-Clarity** [comment]
Could I get a review? @timwright12 @department-of-veterans-affairs/vsa-public-websites-frontend Please and thank you 🙏

**Review by jhonny218 — APPROVED** [review]

**Review by timwright12 — APPROVED** [review]

**Pull request merged**